File: codingTest/programmers/python/소수 만들기.py
import itertools
def isPrime(num):
    if num == 2: return True
    for i in range(2, int(num ** 0.5) + 1):
        if num % i == 0: return False
    return True

def solution(nums):
    count = 0
    for com in itertools.combinations(nums, 3):
        if isPrime(sum(com)): count += 1
    return count

File: codingTest/programmers/javascript/소수 만들기.js
function isPrime(num) {
  for (let i = 2; i < Math.sqrt(num) + 1; i++) if (!(num % i)) return false;
  return true;
}

function solution(nums) {
  let count = 0;
  const size = nums.length;
  for (let i = 0; i < size - 2; i++) {
    for (let j = i + 1; j < size - 1; j++) {
      for (let k = j + 1; k < size; k++) {
        if (isPrime(nums[i] + nums[j] + nums[k])) count++;
      }
    }
  }
  return count;
}
